sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -98,6 +102,12 @@
     @Test
     abstract void recognizeEntitiesForFaultyText(HttpClient httpClient, TextAnalyticsServiceVersion serviceVersion);
 
+    @Test
```

Comment:
I'm not seeing tests for the error that gets thrown when too many documents are being passed (might just not have found the tests). I have tests for every endpoint except `detect_language` and they're all called `test_too_many_documents`. [Here's](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/tests/test_extract_key_phrases.py#L96) an example of one. Even though we're no longer throwing a special error ourselves in this case, I think it's still good to have a test for it

Reply:
I was thinking to do it later when resolving this issue https://github.com/Azure/azure-sdk-for-java/issues/13960. But it should be good to include the tests but keep the issue open until service resolve it.

Reply:
seems only detect_language can take more than 31 documents currently. Does it happen in Python? Other endpoints can throw errors in 10 documents.

Reply:
the limit is different for the different endpoints, see [here](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/concepts/data-limits?tabs=version-3).  For `detect_language` i don't have any tests, since if I have more than 1000 documents, I run into batch oversize error instead. The other tests I do based off of the size limit + 1

Reply:
OK. then i would have no test for detect_language endpoint then